fv/containers/containers.go

```diff
@@ -24,21 +24,38 @@ import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	"github.com/projectcalico/felix/fv/utils"
+	"github.com/projectcalico/libcalico-go/lib/set"
 	log "github.com/sirupsen/logrus"
 )
 
 type Container struct {
 	Name     string
 	IP       string
 	Hostname string
-	Stop     func()
-	stopped  bool
+	runCmd   *exec.Cmd
+	binaries set.Set
 }
 
 var containerIdx = 0
 
 var runningContainers = []*Container{}
 
+func Stop(c *Container) {
+	if c == nil {
+		log.Info("Stop no-op because nil container")
```

Comment:
I think the go idiom is to do an early return here and below (rather than an if/elseif/else.